services/web/server/src/simcore_service_webserver/catalog_api_handlers.py

```diff
@@ -191,7 +242,7 @@ async def get_service_output_handler(request: Request):
     )
 
     # format response
-    enveloped: str = json_dumps({"data": response_model.dict(by_alias=True)})
+    enveloped: str = json_dumps({"data": response_model.dict(**RESPONSE_MODEL_POLICY)})
```

Comment:
ah so you still need to pass it in... I thought it could be set as a global.

Reply:
This policy for the moment is used
- catalog app-module in the webserver, where pydantic is used to build the response model 
- catalog *service*, which is based in fastapi

IN the first case, is used everytime ``dict(*)`` model exporter is used, the the latetter needs to be passed to the route decorator arguments.

It is actually GOOD that these are not global. It makes the whole more testable this way

Reply:
changed